charts/argo-cd/README.md

```diff
@@ -34,6 +34,34 @@ Changes in the `CustomResourceDefinition` resources shall be fixed easily by cop
 
 ## Upgrading
 
+### 3.8.1
+
+This bugfix version potentially introduces a rename (and recreation) of one or more ServiceAccounts. It _only happens_ when you use one of these customization:
+
+```yaml
+# Case 1)
+controller:
+  serviceAccount:
+    name: "" # or <nil>
+
+# Case 2)
+dex:
+  serviceAccount:
+    name: "" # or <nil>
+
+# Case 3)
+server:
+  serviceAccount:
+    name: "" # or <nil>
+
+# Case 4)
+repoServer:
+  serviceAccount:
+    create: true
```

Comment:
1. it would be nice to mention somehow that 4. happens only with `name` not set.
2. why not move this case to the top?

all the rest looks good for me

Reply:
oh, one more thing. it may be worth documenting a workaround to make affected configurations work (in case the users want to preserve old name). I believe, all affected cases will default to `name: argo-cd-argocd`, so `name: argo-cd-argocd` in all cases should revert the new chart to old behaviour.

Reply:
The old name is not that deterministic. It depends on the helm release name. If you installed argocd with `helm install my-release argo/argo-cd` the name was `my-release-argo-cd`